.ci/hack/component_descriptor

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/8490de8f362951b0b34730a6a32c386852b23dee/.ci/component_descriptor#L7-L9: there is one TODO can be addressed with this vendoring.